nuttx/arch/arm/src/stm32/Kconfig

```diff
@@ -1235,156 +1926,166 @@ if STM32_ADC1
 menu "ADC1 Device Initialization"
 
 config STM32_ADC1_CHAN0_INIT
-	bool "ADC1 Channel 0"
+	bool "ADC1 Channel 0 (IN1-PC0)"
```

Comment:
Hi CaveMike,
     This section relates to ADC1, subsequent sections relate to ADC2 and ADC3.   PC0 is configurable for all ADC channels as IN1.
Port	ADC
PA0	ADC12_IN5
PA1	ADC12_IN6
PA2	ADC12_IN7
PA3	ADC12_IN8
PA4	ADC12_ IN9
PA5	ADC12_ IN10
PA6	ADC12_ IN11
PA7	ADC12_ IN12
PB0	ADC12_ IN15
PB1	ADC12_ IN16
PC0	ADC123_ IN1
PC1	ADC123_ IN2
PC2	ADC123_ IN3
PC3	ADC123_ IN4
PC4	ADC12_ IN13
PC5	ADC12_ IN14
PF3	ADC3_IN6
PF4	ADC3_IN7
PF5	ADC3_IN8
PF6	ADC3_IN9
PF7	ADC3_IN10
PF8	ADC3_IN11
PF9	ADC3_IN12
PF10	ADC3_IN13